app/javascript/css/enrollment.scss

```diff
@@ -169,4 +170,33 @@
         }
       }
     }
+
+    .enrollment-details{
```

Comment:
seems to be we're setting some grid and width styles, any reason we can't use bootstrap classes in the view to achieve some of this?

Reply:
not all the necesary styling had bootstrap class so i prefer it all to be in one place rather than half bootstrap classes/half in a style sheet.